src/test/PDFTest/CrudPDFServiceTest.java

```diff
@@ -87,22 +87,89 @@ public void uploadFormTest() {
   //  }
 
   @Test
-  public void uploadImageToPDFTest() {
+  public void uploadImageJPGToPDFTest() {
     User user =
-        createUser()
-            .withUserType(UserType.Admin)
-            .withUsername(username)
-            .withPasswordToHash(password)
-            .buildAndPersist(userDao);
+      createUser()
+        .withUserType(UserType.Client)
+        .withUsername(username)
+        .withPasswordToHash(password)
+        .buildAndPersist(userDao);
     TestUtils.login(username, password);
 
-    File file = new File(resourcesFolderPath + File.separator + "1.png");
+    File file = new File(resourcesFolderPath + File.separator + "veteran-id-card-vic.jpg");
     HttpResponse<String> uploadResponse =
-        Unirest.post(TestUtils.getServerUrl() + "/upload")
-            .header("Content-Disposition", "attachment")
-            .field("pdfType", PDFType.IDENTIFICATION_DOCUMENT)
-            .field("file", file)
-            .asString();
+            Unirest.post(TestUtils.getServerUrl() + "/upload")
+                    .field("pdfType", "IDENTIFICATION_DOCUMENT")
+                    .header("Content-Disposition", "attachment")
+                    .field("file", file)
+                    .asString();
+
+    JSONObject uploadResponseJSON = TestUtils.responseStringToJSON(uploadResponse.getBody());
+    assertThat(uploadResponseJSON.getString("status")).isEqualTo("SUCCESS");
+  }
+
+  @Test
+  public void uploadImagePNGToPDFTest() {
```

Comment:
you need to put the image to pdf service tests in its own test file called ImageToPdfServiceIntegrationTests
you also need to add unit tests, these are integration tests

Reply:
Done